Gemfile

```diff
@@ -43,8 +43,7 @@ group :system_tests do
 
   gem 'beaker-rspec'
   gem 'beaker-windows'
-  # Need at least 85ebe31 for EL9 support
-  gem 'net-ssh', :git => "https://github.com/net-ssh/net-ssh", :ref => 'v7.0.0.beta1'
+  gem 'net-ssh', '> 7.0' # Need 7.0+ for EL9 support
```

Comment:
It surprises me beaker has an open ended dependency. I'd be open to changing beaker's dependency to `~> 7.0` so individual projects don't have to.

Reply:
@ekohl We can't do that unfortunately. Puppet LTS uses Ruby 2.5 which `net-ssh` 7+ doesn't support :-(.

Reply:
I mean, I guess we could if we can detect the version of ruby at runtime and base it on that.

Reply:
Hmm, let's not go that route then. Luckily the dependency is open ended so it can install on modern Rubies and `bundle update` should pick it up anyway.